Mage.Sets/src/mage/cards/l/Lhurgoyf.java

```diff
@@ -33,7 +41,7 @@ public Lhurgoyf(UUID ownerId, CardSetInfo setInfo) {
         this.toughness = new MageInt(0);
 
         // Lhurgoyf's power is equal to the number of creature cards in all graveyards and its toughness is equal to that number plus 1.
-        this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new LhurgoyfEffect()));
+        this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new SetPowerToughnessSourceEffect(powerValue, toughnessValue, Duration.WhileOnBattlefield, SubLayer.CharacteristicDefining_7a, false)));
```

Comment:
This ability needs to be Zone.ALL (not your fault, it was wrong before) and Duration.EndOfGame. It also now needs a `setText` override, since the text generation isn't smart enough to come up with "that number plus 1".